app/Resources/translations/default/ShopThemeCatalog.xlf

```diff
@@ -107,6 +107,15 @@
       </trans-unit>
     </body>
   </file>
+  <file original="modules/ps_facetedsearch/views/templates/front/catalog/active-filters.tpl" source-language="en-US" target-language="en" datatype="plaintext">
```

Comment:
Can't remember why it is in the catalog?

Reply:
Below the extra space at the end has been removed. :-)

Reply:
Yes because it has been modified in the other template So when the extractor parsed the wordings it moved this one in the first template matching (which is now this one) And updated the line matching the other origin file
But it was just moved inside the same file in the end, the trans-unit id is still the same so no problem here